recipes/qt-multimedia/meta.yaml

```
@@ -0,0 +1,115 @@
+build:
+  number: 0
+  detect_binary_files_with_prefix: true
+  run_exports:
+    - {{ pin_subpackage('qt6-multimedia', max_pin='x.x') }}
+  missing_dso_whitelist:  # [win]
+    - '$RPATH/api-ms-win-core-libraryloader-l1-2-0.dll'  # [win]
+    - '$RPATH/api-ms-win-ntuser-sysparams-l1-1-0.dll'  # [win]
+    - '$RPATH/api-ms-win-core-heap-l2-1-0.dll'  # [win]
+    - '$RPATH/api-ms-win-core-libraryloader-l1-2-1.dll'  # [win]
+
+requirements:
+  build:
+    - make                               # [unix]
+    - cmake
+    - ninja
+    - perl
+    - ruby >=2.5                         # [linux]
+    - bison                              # [linux]
+    - flex                               # [linux]
+    - gperf                              # [linux]
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+    - {{ stdlib("c") }}
+    - qt6-main {{ version }}             # [build_platform != target_platform]
+    - {{ cdt('mesa-libgl-devel') }}      # [linux]
+    - {{ cdt('mesa-libegl-devel') }}     # [linux]
+    - {{ cdt('mesa-dri-drivers') }}      # [linux]
+    - {{ cdt('libdrm-devel') }}          # [linux]
+    - {{ cdt('libglvnd-glx') }}          # [linux]
+    - {{ cdt('libglvnd-egl') }}          # [linux]
+    - {{ cdt('libglvnd-devel') }}        # [linux]
+
+    # native libs
+    - ffmpeg                             # [build_platform != target_platform]
+    - ffmpeg *=lgpl*                     # [build_platform != target_platform]
+    - gstreamer                          # [build_platform != target_platform and linux]
+    - gst-plugins-base {{ gstreamer }}   # [build_platform != target_platform and linux]
+    - pulseaudio                         # [build_platform != target_platform and linux]
+
+    - xorg-libx11                       # [build_platform != target_platform and linux]
+    - xorg-libxext                      # [build_platform != target_platform and linux]
+    - xorg-libxdamage                   # [build_platform != target_platform and linux]
+    - xorg-libxrandr                    # [build_platform != target_platform and linux]
+    - xorg-libxcomposite                # [build_platform != target_platform and linux]
+    - xorg-libxcursor                   # [build_platform != target_platform and linux]
+    - xorg-libsm                        # [build_platform != target_platform and linux]
+    - xorg-libice                       # [build_platform != target_platform and linux]
+    - xorg-libxxf86vm                   # [build_platform != target_platform and linux]
+    - xorg-libxtst                      # [build_platform != target_platform and linux]
+    - xorg-xf86vidmodeproto             # [build_platform != target_platform and linux]
+  host:
+    - alsa-lib                           # [linux]
+    - ffmpeg
+    - ffmpeg *=lgpl*
+    - gstreamer                          # [unix]
+    - gst-plugins-base  {{ gstreamer }}  # [unix]
+    # gstreamer plugin not supported on win32: https://bugreports.qt.io/browse/QTBUG-107073
```

Please put comments above the dependency they describe
```suggestion
    # gstreamer plugin not supported on win32: https://bugreports.qt.io/browse/QTBUG-107073
    - gstreamer                          # [unix]
    - gst-plugins-base  {{ gstreamer }}  # [unix]
```